From 2c9675bd3040051750dffc61ce0c109d7195ee63 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 18 Sep 2021 14:39:11 +0200
Subject: [PATCH] Makefile: Add DESTDIR

---
 Makefile.in | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 5e8b6fd..9d6a4f3 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -60,12 +60,12 @@ distclean: clean
 	$(CC) $(CFLAGS) $(INCS) -c $< -o $@
 
 install: otf2bdf
-	$(MKINSTALLDIRS) $(bindir) $(mandir)/man1
-	$(CP) otf2bdf $(bindir)/otf2bdf
-	$(CP) otf2bdf.man $(mandir)/man1/otf2bdf.1
+	$(MKINSTALLDIRS) $(DESTDIR)$(bindir) $(DESTDIR)$(mandir)/man1
+	$(CP) otf2bdf $(DESTDIR)$(bindir)/otf2bdf
+	$(CP) otf2bdf.man $(DESTDIR)$(mandir)/man1/otf2bdf.1
 
 uninstall:
-	$(RM) -f $(bindir)/otf2bdf
-	$(RM) -f $(mandir)/man1/otf2bdf.1
+	$(RM) -f $(DESTDIR)$(bindir)/otf2bdf
+	$(RM) -f $(DESTDIR)$(mandir)/man1/otf2bdf.1
 
 # end of Makefile
-- 
2.52.0

