diff --git a/configure.ac b/configure.ac
index 522ad93..661f466 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,8 +40,8 @@ ifdef ENABLE_CROSSBUILD
     hwdbdir=$(TARGET_UDEV)/hwdb.d
 endif
 else
-    udevrulesdir=$(libdir)/udev/rules.d
-    hwdbdir=$(libdir)/udev/hwdb.d
+    udevrulesdir=$(UDEV)/rules.d
+    hwdbdir=$(UDEV)/hwdb.d
 endif
 '
 AC_SUBST([UDEVdata_SNIPPET])
@@ -53,7 +53,7 @@ ifdef ENABLE_CROSSBUILD
     mtp_probedir=$(TARGET_UDEV)
 endif
 else
-    mtp_probedir=$(libdir)/udev
+    mtp_probedir=$(UDEV)
 endif
 '
 AC_SUBST([UDEVbin_SNIPPET])
