diff --git a/test/Checkit b/test/Checkit
index b06083b..48f3459 100644
--- a/test/Checkit
+++ b/test/Checkit
@@ -4,8 +4,9 @@ CC=${CC:-cc}
 echo
 echo "=== +++++++++ This test runs in 15 seconds on a PIII 750Mhz +++++++"
 
-LIBJUDY=`find ../ -name libJudy.a`
+LIBJUDY=`find ../ -name libJudy.so`
 JUDY_H=`find ../ -name Judy.h`
+export LD_LIBRARY_PATH=`dirname $LIBJUDY`
 
 echo "$CC -O SLcompare.c -DHASHMETHOD -o SL_Hash"
       $CC -O SLcompare.c -DHASHMETHOD -o SL_Hash
