diff --git a/third_party/gtest/googletest/src/gtest-death-test.cc b/third_party/gtest/googletest/src/gtest-death-test.cc
index da09a1c..3f76eca 100644
--- a/third_party/gtest/googletest/src/gtest-death-test.cc
+++ b/third_party/gtest/googletest/src/gtest-death-test.cc
@@ -46,7 +46,7 @@
 # include <errno.h>
 # include <fcntl.h>
 # include <limits.h>
-
+# include <cstdint>
 # if GTEST_OS_LINUX
 #  include <signal.h>
 # endif  // GTEST_OS_LINUX
