diff --git a/test/run-tests.py b/test/run-tests.py
index fe1bd72..5a32247 100755
--- a/test/run-tests.py
+++ b/test/run-tests.py
@@ -958,7 +958,7 @@ def main(args):
         # By default, exclude wasi tests because WASI support is not include
         # by int the build by default.
         # TODO(sbc): Find some way to detect the WASI support.
-        exclude_dirs += ['wasi']
+        exclude_dirs += ['wasi', 'wasm2c']
 
     test_names = FindTestFiles('.txt', pattern_re, exclude_dirs)
 
