diff --git a/autogen.sh b/autogen.sh
index 0d014dc..ba08c36 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -1,7 +1,6 @@
 #!/bin/sh
 set -x -e
 mkdir -p admin m4
-gtkdocize --docdir docs/reference
 libtoolize --force
 autopoint -f
 aclocal -Wall -I m4
diff --git a/configure.ac b/configure.ac
index 0bd4a67..d4a8200 100644
--- a/configure.ac
+++ b/configure.ac
@@ -169,7 +169,8 @@ if test "$GCC" = yes ; then
      AC_MSG_RESULT([$enable_Werror])
 fi
 
-GTK_DOC_CHECK
+# GTK_DOC_CHECK
+AM_CONDITIONAL([ENABLE_GTK_DOC], [false])
 
 AC_CONFIG_FILES([Makefile po/Makefile.in docs/Makefile docs/reference/Makefile
 libuser.pc])
diff --git a/docs/Makefile.am b/docs/Makefile.am
index daa2288..aaeb9a0 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -8,16 +8,18 @@ CLEANFILES = $(man_MANS) $(SGML_OUTPUTS)
 EXTRA_DIST = attributes.txt libuser.conf.5.in rfc2307.txt sgml/libuser.sgml \
 	$(SGML_OUTPUTS)
 
-all: sgml/libuser.txt sgml/libuser.html
+# Disable SGML documentation generation
+# all: sgml/libuser.txt sgml/libuser.html
+all:
 
 libuser.conf.5: $(srcdir)/libuser.conf.5.in Makefile
 	sed 's,@sysconfdir\@,$(sysconfdir),g' \
 		< $(srcdir)/libuser.conf.5.in > $@
 
-sgml/libuser.txt: $(abs_srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2txt $(abs_srcdir)/sgml/libuser.sgml
+# sgml/libuser.txt: $(abs_srcdir)/sgml/libuser.sgml
+# 	[ -d sgml ] || mkdir sgml
+# 	cd sgml; sgml2txt $(abs_srcdir)/sgml/libuser.sgml
 
-sgml/libuser.html: $(abs_srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2html $(abs_srcdir)/sgml/libuser.sgml
+# sgml/libuser.html: $(abs_srcdir)/sgml/libuser.sgml
+# 	[ -d sgml ] || mkdir sgml
+# 	cd sgml; sgml2html $(abs_srcdir)/sgml/libuser.sgml
diff --git a/docs/reference/Makefile.am b/docs/reference/Makefile.am
index 89724c9..f2e87ef 100644
--- a/docs/reference/Makefile.am
+++ b/docs/reference/Makefile.am
@@ -73,7 +73,7 @@ GTKDOC_CFLAGS=
 GTKDOC_LIBS=
 
 # This includes the standard gtk-doc make rules, copied by gtkdocize.
-include $(srcdir)/gtk-doc.make
+# include $(srcdir)/gtk-doc.make
 
 # Comment this out if you want 'make check' to test you doc status
 # and run some sanity checks
