diff --git a/Makefile b/Makefile
index c5544b1..1583a27 100644
--- a/Makefile
+++ b/Makefile
@@ -1,20 +1,20 @@
-prefix=/usr/local
-bindir=$(prefix)/bin
-libdir=$(prefix)/lib
-mandir=$(prefix)/man
-rpmdumpheader=$(bindir)/rpmdumpheader
+prefix?=/usr/local
+bindir?=$(prefix)/bin
+libdir?=$(prefix)/lib
+mandir?=$(prefix)/man
+rpmdumpheader?=$(bindir)/rpmdumpheader
 zlibdir=zlib-1.2.2.f-rsyncable
 zlibbundled=$(zlibdir)/libz.a
 zlibldflags=$(zlibbundled)
 zlibcppflags=-I$(zlibdir)
-pylibprefix=/
+pylibprefix?=/
 CFLAGS = -fPIC -O2 -Wall -g -DWITH_ZSTD=1
 CPPFLAGS = -fPIC -DDELTARPM_64BIT -DBSDIFF_NO_SUF -DRPMDUMPHEADER=\"$(rpmdumpheader)\" $(zlibcppflags)
 LDLIBS = -lbz2 $(zlibldflags) -llzma -lzstd
 LDFLAGS =
 PYTHONS = python python3
 
-all: makedeltarpm applydeltarpm rpmdumpheader makedeltaiso applydeltaiso combinedeltarpm fragiso
+all: makedeltarpm applydeltarpm rpmdumpheader makedeltaiso applydeltaiso combinedeltarpm fragiso python
 
 python: _deltarpmmodule.so
 
